django/test/testcases.py

```diff
@@ -821,6 +821,15 @@ class TransactionTestCase(SimpleTestCase):
     # This can be slow; this flag allows enabling on a per-case basis.
     serialized_rollback = False
 
+    # This attribute is strongly linked to serialized_rollback parameter.
+    # It allows the data restoration after the database flush, at the end of
+    # the test, if the next test needs these initial data.
+    # It is updated by the test runner when it builds the test suite.
+    # Being initialised to True is really important: the last
+    # TransactionTestCase, which has no following tests with
+    # serialized_rollback option, will always be set to True.
+    _next_serialized_rollback = True
```

Comment:
Not sure that's a good idea to add a dependency between the default value here and the logic behind proper `serialized_rollback` initialization. Probably better to add more logic on `reorder_postprocess`.